gslib/tests/test_wildcard_iterator.py

```diff
@@ -284,6 +306,64 @@ def testDoesNotStripDelimiterForDoubleWildcard(self):
     self.assertEqual(expected_prefixes, actual_prefixes)
     self.assertEqual(expected_uri_strs, actual_uri_strs)
 
+  def testDoubleWildcardAfterBucket(self):
+    """Tests gs://bucket/**/object matching."""
+    actual_uri_strs = set()
+    actual_prefixes = set()
+    for blr in self._test_wildcard_iterator(
```

Comment:
What is "blr"? Is there a more clear name?

Reply:
It stands for BucketListingRef. I know it's not a great name, but is used widely in gsutil, so I will leave it like that for now.